manifest.json

```
@@ -0,0 +1,25 @@
+{
+  "vendor": "vtex",
+  "name": "wishlist",
+  "title": "Wishlist",
+  "version": "0.0.0",
+  "defaultLocale": "pt-BR",
+  "builders": {
+    "pages": "0.x",
+    "react": "2.x"
+  },
+  "mustUpdateAt": "2019-04-20",
+  "categories": [],
+  "registries": [
+    "smartcheckout"
+  ],
+  "settingsSchema": {},
+  "scripts": {
+    "postreleasy": "vtex publish --verbose"
+  },
+  "dependencies": {
+    "vtex.store-components": "2.x",
+    "vtex.styleguide": "6.x",
```

Upgrade styleguide version
```suggestion
    "vtex.styleguide": "7.x",
```